cypress.json

```diff
@@ -0,0 +1,11 @@
+{
+  "baseUrl": "http://127.0.0.1:8080",
```

Comment:
Reviewing #35, I noticed that this `baseUrl` is not needed because:

* If `--url` option is not set, `yarn test:e2e` lanuches a dev server with the baseUrl which is automatically determined; you don't need `yarn serve` separatedly.
* The baseUrl is accessible from test codes with `Cypress.config('baseUrl')`, not with `URL` or `Cypress.env('URL')`.
* In case you want to fix the baseUrl, you can manually set it with `--url` option.

So, for this reason, I think that

* `--url=...` in package.json
* `URL` in tests/e2e/config.js

are not needed, too.

What do you think? 🤔

Reply:
I don't know why, but my PC can't run without `--url` option. So for that reason I added this option to package.json, I will remove it. About `URL` property it's good decision. I will remove it.

Also config.js removed in PR #35. There I added env that should be helpful for CI.

Reply:
> I don't know why, but my PC can't run without --url option.

I was troubled with the same situation quite a while ago, too. In my case, it was because of my machine's CPU resource; Docker-for-Mac ate so great part of the resource that a dev server couldn't start. 🐷
If there is no problem, Cypress's Vue plugin is supposed to lauch a dev server at this line, though.
https://github.com/vuejs/vue-cli/blob/dev/packages/%40vue/cli-plugin-e2e-cypress/index.js#L26